FWCore/Framework/src/StreamSchedule.cc

```diff
@@ -565,67 +688,7 @@ namespace edm {
       conditionalmods = std::unordered_set<std::string>(
           std::make_move_iterator(itCondBegin + 1), std::make_move_iterator(modnames.begin() + modnames.size() - 1));
 
-      for (auto const& cond : conditionalmods) {
```

Comment:
This chunk of code has been moved exactly to ConditionalTaskHelper (with some code now being in its own functions).